travis/build-all-in-one-image.sh

```diff
@@ -10,7 +10,7 @@ echo "TRAVIS_BRANCH=$TRAVIS_BRANCH, REPO=$REPO, PR=$PR, BRANCH=$BRANCH, TAG=$TAG
 make build-all-in-one-linux
 
 # Only push the docker container to Docker Hub for master branch
-if [ "$BRANCH" == "master" ]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
+#if [ "$BRANCH" == "master" ]; then echo 'upload to Docker Hub'; else echo 'skip docker upload for PR'; exit 0; fi
```

Comment:
this is just for testing, gonna upload the container to docker hub to make sure the UI works. Will uncomment later.